### PR TITLE
fix grammatical issues in introduction to events

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description

fix grammatical issues in introduction to events under javascript building blocks.

### CODE REVIEW

File: files/en-us/learn/javascript/building_blocks/events/index.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ First, make a local copy of [random-color-addeventlistener.html](https://github.
 It's just a copy of the simple random color example we've played with already. Now try changing `click` to the following different values in turn, and observing the results in the example:
 
 - [`focus`](/en-US/docs/Web/API/Element/focus_event) and [`blur`](/en-US/docs/Web/API/Element/blur_event) — The color changes when the button is focused and unfocused; try pressing the tab to focus on the button and press the tab again to focus away from the button.
-  These are often used to display information about filling in form fields when they are focused, or displaying an error message if a form field is filled with an incorrect value.
+  These are often used to display information about filling in form fields when they are focused, or display an error message if a form field is filled with an incorrect value.
```
Comment (estelle) [inline files/en-us/learn/javascript/building_blocks/events/index.md:180]:
```suggestion
  These are often used to display information about filling in form fields when they are focused, or to display an error message if a form field is filled with an incorrect value.
```

**Review by estelle — APPROVED** [review]
Thank you!

File: files/en-us/learn/javascript/building_blocks/events/index.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ The example output is as follows. Try clicking the button:
 Events are not unique to JavaScript — most programming languages have some kind of event model, and the way the model works often differs from JavaScript's way.
 In fact, the event model in JavaScript for web pages differs from the event model for JavaScript as it is used in other environments.
 
-For example, [Node.js](/en-US/docs/Learn/Server-side/Express_Nodejs) is a very popular JavaScript runtime that enables developers to use JavaScript to build network and server-side applications.
+For example, [Node.js](/en-US/docs/Learn/Server-side/Express_Nodejs) is a very popular JavaScript runtime that enables developers to use JavaScript to build a network and server-side applications.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/building_blocks/events/index.md:124]: Sorry for being slightly slow to review this, but "network" is a modifier for "applications" here, not a noun of itself.

Comment (yashikabhargava) [reply files/en-us/learn/javascript/building_blocks/events/index.md:124]: No Problem, so I undo the change, right?

Comment (yashikabhargava) [reply files/en-us/learn/javascript/building_blocks/events/index.md:124]: Can you please tell me If I need to create another PR as I'm not able to edit the file?

Comment (Josh-Cena) [reply files/en-us/learn/javascript/building_blocks/events/index.md:124]: Yes, please create another PR to make this change. Thanks!

**Pull request merged**